entities/src/main/java/org/odk/collect/entities/storage/EntitiesRepository.kt

```diff
@@ -5,34 +5,10 @@ import org.odk.collect.shared.Query
 interface EntitiesRepository {
     fun save(list: String, vararg entities: Entity)
     fun getLists(): Set<String>
-
-    @Deprecated(
-        message = "Should use #query instead",
-        ReplaceWith(
-            "query(list, null)"
-        )
-    )
-    fun getEntities(list: String): List<Entity.Saved>
     fun getCount(list: String): Int
     fun addList(list: String)
-    fun delete(id: String)
-    fun query(list: String, query: Query): List<Entity.Saved>
-
-    @Deprecated(
-        message = "Should use #query instead",
-        ReplaceWith(
-            "query(list, Query.Eq(EntitiesTable.COLUMN_ID, id))"
-        )
-    )
-    fun getById(list: String, id: String): Entity.Saved?
-    fun getAllByProperty(list: String, property: String, value: String): List<Entity.Saved>
-
-    @Deprecated(
```

Comment:
Why did you remove this deprecation? Can't we use `query` directly instead of `getByIndex`?